code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -280,7 +280,10 @@
 
 			var/randn = rand(1, 100)
 			last_push_time = world.time
-			if(!(species.flags & NO_SLIP) && randn <= 25)
+			// We ARE wearing shoes OR
+			// We as a species CAN be slipped when barefoot
+			// And also 1 in 4 because rngesus
+			if(shoes || !(species.flags & NO_SLIP) && randn <= 25)

```

Comment:
I might be reading this wrong, but it looks like this will make it so if you're wearing shoes, you'll get disarmed 100% of the time, and if you're not wearing shoes, it will be 25% of the time?

Reply:
... That might explain why I got the disarm on the first try in testing.